### PR TITLE
[SHELLFIND] Fix column sorting. CORE-16825

### PR DESCRIPTION
**Pull request by b-r-o-c-k**
## Purpose

Fix column sorting in search results folder.

JIRA issue: [CORE-16825](https://jira.reactos.org/browse/CORE-16825)

### CODE REVIEW

File: dll/win32/browseui/shellfind/CFindFolder.cpp [hunk 1]
```diff
@@ -538,7 +538,20 @@ STDMETHODIMP CFindFolder::BindToStorage(PCUIDLIST_RELATIVE pidl, LPBC pbcReserve
...
+        break;
+    case 1: // Path
+        return MAKE_COMPARE_HRESULT(StrCmpW(_ILGetPath(pidl1), _ILGetPath(pidl2)));
+    case 2: // Relevance
+        return E_NOTIMPL;
+    default: // Default columns
+        wColumn -= _countof(g_ColumnDefs) - 1;
+        break;
+    }
+    return m_pisfInner->CompareIDs(HIWORD(lParam) | wColumn, _ILGetFSPidl(pidl1), _ILGetFSPidl(pidl2));
```
Comment (KyleKatarn31) [inline dll/win32/browseui/shellfind/CFindFolder.cpp:554]: The basic problem was on File Size... I don't see anything here which would adress the issue. Am I missing something ?

**Comment by KyleKatarn31** [comment]
Why merged ?? Haven't you seen my comment ? This does nothing wrt to the reported issue !

File: dll/win32/browseui/shellfind/CFindFolder.cpp (hunk 1 above)
Comment (b-r-o-c-k) [reply dll/win32/browseui/shellfind/CFindFolder.cpp:554]: File size is a default column that is already handled by the parent shell folder class. The problem was caused by this function not correctly calling the parent `CompareIDs()` function to handle the default columns.

Comment (KyleKatarn31) [reply dll/win32/browseui/shellfind/CFindFolder.cpp:554]: Ok thanks, very clear.

**Pull request merged**